packages/dd-trace/test/appsec/index.spec.js

```diff
@@ -203,5 +204,92 @@ describe('AppSec Index', () => {
       }, context)
       expect(Reporter.finishAttacks).to.have.been.calledOnceWithExactly(req, context)
     })
+
+    it('should propagate incoming http end data with weird framework', () => {
```

Comment:
This should be more descriptive. "Weird framework" doesn't mean much.

Reply:
Well in this case it's trying to describe a framework that has express like properties, but not the right prop types, should I just write that out ?

Reply:
Surely there are better semantics than "weird".

Reply:
```suggestion
    it('should propagate incoming http end data with invalid framework properties', () => {
```

Reply:
wdyt ?